### PR TITLE
Feature/ta27963

### PR DESCRIPTION
**Pull request by itshane**

### CODE REVIEW

File: ZenPacks/zenoss/LinuxMonitor/LinuxService.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+All custom behavior for Linux OS Service is defined here.
+
+"""
+
+
+from . import schema
+
+
+class LinuxService(schema.LinuxService):
```
Comment (itshane) [inline ZenPacks/zenoss/LinuxMonitor/LinuxService.py]: Still a question for me whether to inherit it from OSComponent or use zenpacklib.Component as you can see below

File: ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/os_service.py [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+def systemd_getServices(services):
+    return ''.join(map(lambda x: x or '\n', services)).splitlines()
+
+def systemd_getProcesses(line):
+    matches = RE_PROCESS.findall(line)
+    if matches:
+        return zip(*matches)[0]
+
+
+SERVICE_MAP = {
```
Comment (itshane) [inline ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/os_service.py]: Too lazy to create additional classes.

File: ZenPacks/zenoss/LinuxMonitor/LinuxService.py (hunk 1 above)
Comment (pcarinhas) [inline ZenPacks/zenoss/LinuxMonitor/LinuxService.py]: Normally we start at the Relationship level to see if we need to inherit.
If there is a question about it, fire up a conversation with the lead on this.
If its not broken as is, don't fix it.

File: ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/os_service.py [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+    },
+    'UNKNOWN': None
+}
+
+
+class os_service(LinuxCommandPlugin):
+
+    command = ('export PATH=$PATH:/bin:/sbin:/usr/sbin; '
+               'if command -v systemctl >/dev/null 2>&1; then '
+                'echo "SYSTEMD"; '
```
Comment (itshane) [inline ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/os_service.py]: This one and similar will be a pep8 issue, IMHO, it's much better to have some space here.

**Pull request merged**